src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const MALE = 'm';
+  const men = people.filter(({ sex }) => sex === MALE);
+
+  const menDiedInCentury = century
+    ? men.filter(({ died }) => Math.ceil(died / 100) === century)
+    : men;
```

Comment:
If `century` is false, then you will simply overwrite the `men` array to `menDiedInCentury`, but this will not be completely correct in terms of the name.
```suggestion
  let men = people.filter(({ sex }) => sex === MALE);
  if (century) {
    men = men.filter(({ died }) => Math.ceil(died / 100) === century);
  }
```